src/parsing/static_files.go

```diff
@@ -0,0 +1,116 @@
+package parsing
+
+import (
+	"BGPAlert/common"
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+func ParseStaticFile(folderDir string, msgChannel chan []common.BGPMessage) {
+	// Gets all files from directory we want to test
+	files, err := os.ReadDir("static_data/" + folderDir)
+	if err != nil {
```

Comment:
I think the function should stop here and return an error.

Reply:
yeah agreed. if this fails, probably something more fundamental is failing. best to just absolutely bail out.